packages/dd-trace/src/exporters/latency-stats/writer.js

```diff
@@ -0,0 +1,54 @@
+
+const { SpanStatsEncoder } = require('../../encode/span-stats')
+
+const pkg = require('../../../../../package.json')
+
+const BaseWriter = require('../common/writer')
+const request = require('../common/request')
+const log = require('../../log')
+
+class Writer extends BaseWriter {
```

Comment:
This looks pretty much exactly the same as the stats writer except the URL. Can that code be reused?

Reply:
This is also true for some other parts of this PR.